hail/python/hailtop/gear/deploy_config.py

```diff
@@ -0,0 +1,98 @@
+import os
+import json
+import logging
+from aiohttp import web
+
+log = logging.getLogger('gear')
+
+
+class DeployConfig:
```

Comment:
I think we should have tests for this.

Reply:
I think this is well tested because deploy_config is used by both the clients and the servers.  I added tests for the case of default_namespace=default since that's not actually exercised in the tests (and arguably the most important).